src/main/resources/i18n/messages.properties

```diff
@@ -2853,4 +2853,45 @@ FilterByFile.valid.single=One sample name has been matched to a project.
 FilterByFile.valid.plural={0} sample names have been matched to a project.
 FilterByFile.invalid=The following sample names could not be matched to a project.
 FilterByFile.filter=Filter
-FilterByFile.file-label=Select file containing sample names
\ No newline at end of file
+FilterByFile.file-label=Select file containing sample names
+
+# ========================================================================================== #
+# ForgotPassword Component                                                                   #
+# ========================================================================================== #
+ForgotPassword.credentialsRequired=Username or email is required
+ForgotPassword.input.placeholder=Username or Email Address
+ForgotPassword.button.resetPassword=Reset Password
+ForgotPassword.link.returnToLogin=Return to Login Page
+
+server.ForgotPassword.checkEmail=Check your email for password reset instructions
+server.ForgotPassword.errorSendingInstructions=There was an error sending password reset instructions. Please contact the system administrator
+server.ForgotPassword.emailOrUsernameNotExist=If a user with the provided email address or username exists, you will receive an email with password reset instructions.
+
+# ========================================================================================== #
+# ActivateAccount Component                                                                  #
+# ========================================================================================== #
+ActivateAccount.invalidActivationId=Invalid Activation ID. Please enter a valid activation id.
+ActivateAccount.activationIdRequired=Activation ID is required
+ActivateAccount.input.placeholder=Activation ID
+ActivateAccount.button.activateAccount=Activate Account
+ActivateAccount.link.returnToLogin=Return to Login Page
+
+# ========================================================================================== #
+# PasswordReset Component                                                                    #
+# ========================================================================================== #
+PasswordReset.title=Reset Password
+PasswordReset.alert.title=Password Policy
+PasswordReset.alert.description=Please choose a strong password that meets the following criteria:
+PasswordReset.alert.rule1=Must not be empty.
+PasswordReset.alert.rule2=Must be at least 8 characters long.
```

Comment:
These password criteria suck.  I think we should update it to be just length.

Reply:
Leaving as discussed :)

Reply:
:)